tutorials/commands/flux-job-cancel.rst

```diff
@@ -0,0 +1,119 @@
+option to ``flux mini submit`` that will submit 100 duplicate copies of the ``sleep`` job.
+
+.. code-block:: console
+
+    $ flux mini submit --cc=1-100 sleep inf
+    <snip - many job ids printed out>
+
+    $ flux jobs
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+        ƒjTWS5m3 achu     sleep       S      1      -        -
+        ƒjTWS5m4 achu     sleep       S      1      -        -
+        ƒjTWS5m5 achu     sleep       S      1      -        -
+        ƒjTWS5m6 achu     sleep       S      1      -        -
+	<snip - there are many jobs waiting to be run>
+        ƒjTWS5m2 achu     sleep       R      1      1   8.858s corona212
+        ƒjTWS5m1 achu     sleep       R      1      1   8.860s corona212
+        ƒjTUx6Um achu     sleep       R      1      1   8.870s corona212
+        ƒjTUx6Uk achu     sleep       R      1      1   8.870s corona212
+        ƒjTUx6Uj achu     sleep       R      1      1   8.870s corona212
+        ƒjTUx6Ui achu     sleep       R      1      1   8.871s corona212
+	<snip - there are many jobs running>
+
+As you can see, we have a lot of jobs waiting to run (state ``S``) and a lot of running jobs (state ``R``).
+
+Lets first ``flux job cancelall`` without any options.
+
+.. code-block:: console
+
+    $ flux job cancelall
+    flux-job: Command matched 100 jobs (-f to confirm)
+
+As you can see, ``flux job cancelall`` found all 100 jobs to cancel, but it hasn't canceled them yet.  In order to go through
+with the cancellation you must specify the ``-f`` (or ``--force``) option.
+
+.. code-block:: console
+
+    $ flux job cancelall -f
+    flux-job: Canceled 100 jobs (0 errors)
+
+    $ flux jobs
+               JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+
+As you can see all the jobs are now canceled after passing the ``-f`` option to ``flux job cancelall``.  ``flux jobs``
+confirms this as no jobs are now listed.
+
+``flux job cancellall`` has several options to filter the jobs to cancel.  Perhaps the most commonly used
+option is the ``-S`` or ``--states`` option.  The ``--states`` option specifies the state(s) of a job to cancel.  The most
+common states to target are ``pending`` and ``running``.  Lets resubmit our 100 jobs and see the result
+of trying to cancel ``pending`` vs ``running`` jobs.
+
+.. code-block:: console
+
+    $ flux mini submit --cc=1-100 sleep inf
+    <snip - many job ids printed out>
+
+    $ flux job cancelall --states=pending
+    flux-job: Command matched 52 jobs (-f to confirm)
+
+    $ flux job cancelall --states=running
+    flux-job: Command matched 48 jobs (-f to confirm)
+
+As you can see ``flux job cancelall --states=pending`` would target the 52 pending jobs for cancellation and
+``flux job cancelall --states=running`` would target the current 48 running jobs for cancellation.
+
+And that's it! If you have any questions, please
```

Comment:
Since cancelall is mentioned, might as well mention `flux pkill`?

Reply:
i thought about that, but `pkill` is really about signaling.  So its a bit different?

Reply:
I didn't know Flux resorted to violence. :scream:

Reply:
No `pkill` cancels jobs

Reply:
well crud, i guess i was just thinking `pkill(1)` ... yeah, i guess it should go there too